lib/pbench/test/unit/server/test_controllers_list.py

```diff
@@ -60,11 +62,44 @@ def build_index(self, server_config, dates):
             index += f"{idx}{d},"
         return index
 
-    def test_missing_json_object(self, client, server_config):
+    def test_missing_auth_header(self, client, server_config):
         """
-        test_missing_json_object Test behavior when no JSON payload is given
+        test_missing_json_object Test behavior when no Authorization header is given
         """
         response = client.post(f"{server_config.rest_uri}/controllers/list")
+        assert response.status_code == 401
+
+    def test_malformed_auth_header(self, client, server_config, pbench_token):
+        """
+        test_missing_json_object Test behavior when Authorization header is malformed
+        """
+        response = client.post(
+            f"{server_config.rest_uri}/controllers/list",
+            headers={"Authorization": pbench_token},
+        )
+        assert response.status_code == 401
+
+    def test_non_accessible_user_data(self, client, server_config, pbench_token):
+        """
+        test_missing_json_object Test behavior when Authorization header does not have access to other user's data
+        """
+        # Auth token belongs to the user "drb"
+        # Trying to access the data belong to the user "pp"
```

Comment:
Right, and I should mention this in the comment that the auth token belongs to the "drb" user because the pytest fixture `pbench_token` creates it for the "drb" user. 

This test is just testing whether a different user's auth token can access some other user data (doesn't matter if the data for the other user is present or not, we just won't proceed with the request), test to see if the user can access their own data via auth token is performed in other functions such as `test_query` if thats what you mean